[Docs] Tidy Vizro-ai docs (#466)
mckinsey/vizro: vizro-ai/docs/pages/tutorials/quickstart.md

```python
vizro_ai = VizroAI()
```

> For customization of `VizroAI` class checkout out our guide on [how to customize models](../user-guides/customize-vizro-ai.md).

Reviewer: ```suggestion
For customization of the `VizroAI` class checkout out our guide on [how to customize models](../user-guides/customize-vizro-ai.md).
```

> For customization of the `VizroAI` class checkout out our guide on [how to customize models](../user-guides/customize-vizro-ai.md).

Reviewer 2: ```suggestion
To learn how to customize the `VizroAI` class, check out the guide on [how to customize models](../user-guides/customize-vizro-ai.md).
```
